lib/ClangImporter/ClangImporter.cpp

```diff
@@ -345,6 +344,10 @@ getNormalInvocationArguments(std::vector<std::string> &invocationArgStrs,
           SHIMS_INCLUDE_FLAG, searchPathOpts.RuntimeResourcePath,
       });
 
+  if (!importerOpts.DisableModulesValidateSystemHeaders) {
+    invocationArgStrs.push_back("-fmodules-validate-system-headers");
+  }
```

Comment:
Nitpick: Can you move this down to the end of the function where the other importerOpts options are checked?